pkg/ensurance/collector/collector.go

```diff
@@ -216,6 +235,12 @@ func (s *StateCollector) UpdateCollectors() {
 			}
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.CraneNodeResourceTopology) {
+		if _, exists := s.collectors.Load(types.NodeResourceTopologyCollectorType); !exists {
+			s.collectors.Store(types.NodeResourceTopologyCollectorType,
+				noderesourcetopology.NewNodeResourceTopology(s.nodeName, s.sysPath, s.nrtLister, s.nodeLister, s.kubeClient, s.craneClient))
```

Comment:
do you want to run this code repeatedly with a 10s interval? NewNodeResourceTopology is a one time thing, maybe can be moved to the caller and only trigger once?

Reply:
Got it.